### PR TITLE
Implement connectivity-related setup configuration steps with new API

### LINKED ISSUE #1475: django-set-upconfiguration(Django Admin OIDC) for open inwoner
Kan opgenomen worden in helm charts zegt Sidney.

### PR DESCRIPTION
**Pull request by swrichards**
Closes:

- #1475 
- #1467 
- #1468 (eSuite, not OpenKlant2 yet)

### CODE REVIEW

File: src/open_inwoner/configurations/bootstrap/zgw.py [hunk 1]
```diff
@@ -1,500 +1,98 @@
...
+    enable_setting = "openzaak_config_enable"
+    namespace = "openzaak_config"
+    config_model = OpenZaakConfigurationModel
 
-    def configure(self):
-        if not self.is_enabled():
-            return
+    def execute(self, model: OpenZaakConfigurationModel):
+        if len(model.api_groups) < 1:
+            raise ConfigurationRunFailed("Configure at least one `api_groups` item")
```
Comment (pi-sigma) [inline src/open_inwoner/configurations/bootstrap/zgw.py:54]: You could mention the configuration step to make clearer which part of the configuration is missing the api groups (unless that's already clear from the exception itself)

**Review by pi-sigma — APPROVED** [review]

File: src/open_inwoner/configurations/bootstrap/zgw.py (hunk 1 above)
Comment (swrichards) [reply src/open_inwoner/configurations/bootstrap/zgw.py:54]: The command output should indicate this.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -208,6 +208,9 @@ jobs:
     name: Check that documentation for configuration steps is up-to-date
     runs-on: ubuntu-latest
 
+    # Disabled while we complete the upgrade to the latest setup-configuration
+    if: false
```
Comment (alextreme) [inline .github/workflows/ci.yml:212]: Hasn't this been done?

**Review by alextreme — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 2 above)
Comment (swrichards) [reply .github/workflows/ci.yml:212]: No not yet, there's some complexities.

**Pull request merged**